docs/reference/redirects.asciidoc

```diff
@@ -637,32 +637,67 @@ See:
 [role="exclude",id="testing"]
 === Testing
 
-This page was deleted.
+This page was deleted. 
+Information about the Java testing framework was removed 
+(https://github.com/elastic/elasticsearch/issues/55257[55257]) from the {es} Reference 
+because it was out of date and erroneously implied that it should be used by application developers.  
+There is an open doc issue (https://github.com/elastic/elasticsearch/issues/55258[#55258]) 
```

Comment:
Should we refer to this as an open doc issue? This implies we'd need to update these redirects if the issue status changes. Also don't know if we need to call it a doc issue.
```suggestion
There is an issue ({es-issue}55258[#55258]) 
```